plugins/reports/plugin.py

```
@@ -0,0 +1,49 @@
+    def migrations_path() -> Path:
+        return Path(__file__).parent / "migrations"
+
+    def on_load(self, ctx: dict[str, Any]) -> None:
+        services = ctx.get("services")
+        try:
+            echo = services.get("powertable.echo")
+            echo("warmup")
+        except Exception:
+            pass
+
+    def register_routes(self, app: FastAPI, router: APIRouter) -> None:
```

Ruff (0.12.2)

Unused method argument: `app`

(ARG002)